formatting-tools/.clang-format

```diff
@@ -55,7 +55,6 @@ IndentCaseLabels: false
 IndentExternBlock: AfterExternBlock
 IndentGotoLabels: true
 IndentPPDirectives: AfterHash
-IndentPragmas: true
```

Comment:
Could you just comment this out and put it to the bottom of the file? To make sure we don't forget about this option.

Reply:
There has been no activity on the patch since January 2021. I would rather assume for now that the option is dead. Therefore, I would not add it commented out, since we don't know if the option will ever come back.

Reply:
That is unfortunate. Thanks for the hint.